blueprints/resource/index.js

```diff
@@ -1,5 +1,5 @@
 var inflection = require('inflection');
-var Promise    = require('ember-cli/lib/ext/promise');
+var RSVP       = require('rsvp');
```

Comment:
I think we get [`rsvp`](https://www.npmjs.com/package/rsvp) through an implicit dependency (probably `ember-cli`), but we should add it as a dependency directly in `package.json`.

Reply:
Thanks, I'll try!

Reply:
(And, thanks so much for supporting this thing!)